README.md

```diff
@@ -20,8 +20,58 @@ docker run -it -p 10300:10300 -v /path/to/local/data:/data rhasspy/wyoming-whisp
 docker run -it -p 10200:10200 -v /path/to/local/data:/data rhasspy/wyoming-piper --voice en_US-lessac-medium
 ```
 
+
 ## Run openWakeWord
 
 ``` sh
 docker run -it -p 10400:10400 rhasspy/wyoming-openwakeword --preload-model 'ok_nabu'
 ```
+
+
+## To run in standalone server
+
+### Run without GPU
+
+Build openwakeword, piper and whisper without GPU with:
+
+``` sh
+docker compose -f docker-compose.base.yml build --no-cache
+```
+
+Run it with:
+
+``` sh
+docker compose -f docker-compose.base.yml up -d
+```
+
+Take it down with:
+
+``` sh
+docker compose down
+```
+
+### Run with GPU
```

Comment:
should we reference documentation on how to setup docker for gpu? (I can of course add it in a seperate pr)

Reply:
Yes, good idea!